.kokoro/grpc/storage.proto

```diff
@@ -0,0 +1,29 @@
+syntax = "proto3";
+
+option java_multiple_files = true;
+option java_package = "grpcstorage";
+option java_outer_classname = "grpcStorage";
+
+package storage_bench;
+
+service storage{
+  rpc Write(ObjectWrite) returns (Completed) {}
+  rpc Read(ObjectRead) returns (Completed){}
+}
+
+message ObjectRead{
+  string bucketName = 1;
+  string objectName = 2;
+}
+
+
+message ObjectWrite{
+  string bucketName = 1;
+  string objectName = 2;
+  string destination = 3;
+}
+
+message Completed{
```

Comment:
If the server panics (log.Fatalf), the client will get the error message. So, thankfully we don't have to handle error propagation ourselves.